charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All changes to this chart will be documented in this file.
 * Improve the description of deprecated `ApplicationNodes.jvmOpts` and `ApplicationNodes.jvmCeOpts` values
 * Run the initSysctl init-container as root to prevent 'permission denied' issues
 * Add revisionHistoryLimit configuration for SonarQube application Deployment ReplicaSets
+* Introduce `ApplicationNodes.podDistributionBudget` and `searchNodes.podDistributionBudget` and deprecate `ApplicationNodes.podDistributionBudget` and `searchNodes.podDistributionBudget`.
```

Comment:
Small typo while deprecating a typo xD 

"Introduce `ApplicationNodes.podDistributionBudget` and `searchNodes.podDistributionBudget`" -> "Introduce `ApplicationNodes.podDisruptionBudget` and `searchNodes.podDisruptionBudget`"